chat-service/src/main/resources/application.yml

```
@@ -44,4 +44,23 @@ eureka:
+      endpoint: ${OTEL_COLLECTOR_TRACES_ENDPOINT}
+    logging:
+      endpoint: ${OTEL_COLLECTOR_LOGS_ENDPOINT}
+    metrics:
+      export:
+        url: ${OTEL_COLLECTOR_METRICS_ENDPOINT}
+
+  # actuator endpoint를 외부에서 볼 수 있도록
+  endpoints:
+    web:
+      exposure:
+        include: "*"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)